src/view/drag-handle/drag-handle.jsx

```diff
@@ -19,19 +20,28 @@ import shouldAllowDraggingFromTarget from './util/should-allow-dragging-from-tar
 import createMouseSensor from './sensor/create-mouse-sensor';
 import createKeyboardSensor from './sensor/create-keyboard-sensor';
 import createTouchSensor from './sensor/create-touch-sensor';
+import createPointerSensor from './sensor/create-pointer-sensor';
 import { warning } from '../../dev-warning';
 
 const preventHtml5Dnd = (event: DragEvent) => {
   event.preventDefault();
 };
 
-type Sensor = MouseSensor | KeyboardSensor | TouchSensor;
+type Sensor = MouseSensor | KeyboardSensor | TouchSensor | PointerSensor;
+
+let IS_TOUCH_COMPATIBLE: boolean = false;
+
+window.addEventListener('touchstart', function onFirstTouch() {
+  IS_TOUCH_COMPATIBLE = true;
```

Comment:
Seems like on a touch compatible system this must happen before any PointerEvent happens. Maybe set something on the first PointerEvent and add an invariant here that checks that has not been set?